src/components/QuizGameApi/index.jsx

```diff
@@ -0,0 +1,14 @@
+// first line to rendomlly picking a question
+const shuffleArray = (array) => [...array].sort(() => Math.random() - 0.5);
+// iam fetching data from a website called opentdb.com this website will give us a simple questions about the nature.
+export const fetchQuizQuestions = async (amount, difficulty) => {
+  const endpoint = `https://opentdb.com/api.php?amount=${amount}&category=17&difficulty=${difficulty}&type=multiple`;
+  const data = await (await fetch(endpoint)).json();
```

Comment:
actually i used this because i remember a case during the class when halit used it and louis said its good idea to use it like this, but if you guys want me to change it i can change

Reply:
Hmm maybe. If they said something like that there must be reason.

Reply:
doesn't really matter either works

Reply:
i don't think i wrote it this way in class because the nested awaits would confuse students. i think i wrote it tarkans way every time i taught it